src/scripts/main.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
-document.addEventListener('click', (e) => {
-  // write code here
-});
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+if (!wall || !spider) {
+  // eslint-disable-next-line no-console
+  console.error('Wall or spider element not found!');
+} else {
+  wall.addEventListener('click', (e) => {
+    const wallCoords = wall.getBoundingClientRect();
+    const height = spider.clientHeight / 2;
+    const width = spider.clientWidth / 2;
```

Comment:
The calculation of `height` and `width` should ensure that the spider's center is exactly under the pointer on click. Currently, it uses `spider.clientHeight / 2` and `spider.clientWidth / 2`, which is correct for centering, but make sure this logic is applied correctly in the positioning calculations below.